pairol_alex/Assignments/Assignment-Stars.py

```diff
@@ -0,0 +1,55 @@
+y = [4,'Tom',1,'Michael',5,7,'Jimmy Smith', 'AlexP']
+def draw_stars(my_list):
+    for item in my_list:
+        output = ''
+        if type(item) is int:
+            for i in range(item):
+                output += '*'
```

Comment:
This works perfect, but in python, you can do output = `item * '*'` rather than using:

> for i in range(item):
>                 output += '*'